src/Connection.js

```diff
@@ -90,6 +90,64 @@ class Connection extends EventEmitter {
 
     }
 
+    readAudio(data) {
+        // Packet format:
+        // https://github.com/mumble-voip/mumble-protocol/blob/master/voice_data.rst#packet-format
+        const audioType = (data[0] & 0xE0) >> 5;
+        const audioTarget = data[0] & 0x1F;
+
+        //console.debug("\nAUDIO DATA length:" + data.length + ' audioType:' + audioType + ' audioTarget: ' + audioTarget);
+
+        if (audioType != Connection.codec().Opus) {
+            // Not OPUS-encoded => not supported :/
+            // Should we warn the user here? Throw an error?
```

Comment:
It's probably a good idea to warn the user here. At least with a `console.warn` or something.